docs/source/support/overview.rst

```diff
@@ -0,0 +1,60 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Ecosystem Overview
+=======================
+
+.. toctree::
+   :hidden:
+
+   status-definitions
+   component-statuses
+
+This page is intended to give you an overview of the components that constitute the DAML Ecosystem, what status they are in, and how they fit together. It lays out DAML's "public API" in the sense of :ref:`Semantic Versioning <versioning>`, and is a prerequisite to understanding DAML's :doc:`compatibility`.
+
+The pages :doc:`status-definitions` and :doc:`component-statuses` give a fine-grained view of what labels like "Alpha" and "Beta" mean, which components expose public APIs and what status they are in.
+
+.. _ecosystem-architecture:
+
+Architecture
+------------
+
+A high level view of the architecture of a DAML application or solution is helpful to make sense of how individual components, APIs and features fit into the DAML Stack.
+
+.. figure:: architecture.png
+
+DAML Networks
+.............
+
+At the bottom of every DAML Application is a DAML Network, a distributed, or possibly centralized persistence infrastructure together with DAML Drivers. DAML Drivers enable the persistence infrastructure to act as a consensus, messaging, and in some cases persistence layer for DAML Applications. Most DAML Drivers will have a public API, but there are no _uniform_ public APIs on DAML Drivers. A good example is the command line interface of `DAML for Postgres <https://github.com/digital-asset/daml/blob/master/ledger/daml-on-sql/README.rst>`_. It's a public interface, but specific to the Postgres driver.
```

Comment:
Is it confusing to newbies to say that most drivers have no standard public API?  Perhaps we want so say what that public API would be used for.  An uninformed implication of the above is that DAML apps are not portable - until you realize that you don't actually code against a driver.  Or is this all clear to a developer?

Reply:
Added a clarifying sentence.